WBooks/Screens/BookLibrary/BookLibraryController.swift

```diff
@@ -22,14 +23,17 @@ class BookLibraryController: UIViewController {
         fatalError("init(nibName:bundle:) has not been implemented")
     }
     
-    var libraryViewModel: BookLibraryViewModel = {
+    var bookDetailsModel: BookLibraryViewModel = {
```

Comment:
This doesn't need to be a computed property. you can have it like this: `let bookLibraryViewModel = BookLibraryViewModel()`